src/script/ptl-tool.py

```diff
@@ -178,9 +178,9 @@
     pass
 GITHUB_API_KEY = os.getenv("PTL_TOOL_GITHUB_API_KEY", GITHUB_API_KEY)
 INDICATIONS = [
-    re.compile("(Reviewed-by: .+ <[\w@.-]+>)", re.IGNORECASE),
-    re.compile("(Acked-by: .+ <[\w@.-]+>)", re.IGNORECASE),
-    re.compile("(Tested-by: .+ <[\w@.-]+>)", re.IGNORECASE),
+    re.compile("(Reviewed-by: .+ <[\\w@.-]+>)", re.IGNORECASE),
+    re.compile("(Acked-by: .+ <[\\w@.-]+>)", re.IGNORECASE),
+    re.compile("(Tested-by: .+ <[\\w@.-]+>)", re.IGNORECASE),
```

Comment:
why not use the raw string, like
```py
re.compile(r"(Tested-by: .+ <[\w@.-]+>)", re.IGNORECASE),
```

simpler and more readable this way.

Reply:
ack
updated PR with the suggestion
thanks Kefu